articles/azure/azs-how-create-load-balancer.md

```diff
@@ -0,0 +1,282 @@
+
+   - **Source port range** - Provide a single port, such as 80, or a port range, such as 1024-65535. This specifies from which ports traffic will be allowed or denied by this rule. Provide an asterisk (*) to allow traffic on any port.
+
+   - **Destination** - The destination filter can be Any, an IP address range, or a default tag. It specifies the outgoing traffic for a specific destination IP address range that will be allowed or denied by this rule.
+
+   - **Destination port range** - Provide a single port, such as 80, or a port range, such as 1024-65535. This specifies from which ports traffic will be allowed or denied by this rule. Provide an asterisk (*) to allow traffic on any port.
+
+   - **Protocol** - Specify whether to allow inbound traffic using UDP, TCP or both.
+
+   - **Action** - Choose whether the rule allows or denies the traffic specified.
+
+   - **Priority** - Rules are processed in priority order; the lower the number, the higher the priority. We recommend leaving gaps between rules - 100, 200, 300, etc. - so that it's easier to add new rules without having to edit existing rules.
+
+   - **Name** - The name of the security rule.
+
+   - **Description** - A description of the security rule.
+  
+    ![Add an inbound security rule](images/azs-browser-add-in-security-rule.png)
+
+6. Click **Add**.
+
+### Create a back-end address pool
+
+To spread traffic load across the VMs, a back-end address pool must be created. This pool contains the IP addresses of the VMs.
+
+1. Navigate to your load balancer by clicking on **All services** in the favourites panel, then selecting **Load balancers** under the networking section.
+
+2. Select the load balancer you created earlier from the list.
+
+3. In the blade for your load balancer, select **Backend pools** under the **Settings** section.
+
+4. Click the **+Add** button.
+
+5. In the **Add backend pool** blade, enter a name for the backend pool.
+
+6. Select **Availability set** for the **Associated to** option, then select the availability set that you created in the previous section.
+
+7. Click the **+ Add a target network IP configuration** button, select the first VM you created in the previous section and select the network IP to be associated with the back end pool.
+
+8. Repeat step 7 for the second VM.
+
+   ![Add a backend pool](images/azs-browser-add-backend-pool.png)
+
+9. Click **OK**
+
+### Create a health probe
+
+When creating a load balancer, a health probe is used to monitor the status of your application. In Azure Stack, the health probe adds and removes VMs from the load balancer backend pool based on their current status.
+
+1. Navigate to your load balancer by clicking on **All services** in the favourites panel, then selecting **Load balancers** under the networking section.
+
+2. Select the load balancer you created earlier from the list.

```

Comment:
From the list, select the load balancer you created earlier.

(does the list have a label that you can use to identify it? If not, maybe provide a screen shot to show which list you mean unless it's really obvious)

Reply:
Fixed. The list fills the entire screen and its a list of created load balancers, so it should be fairly obvious